third_party/xla_client/xrt_computation_client.cc

```diff
@@ -261,7 +262,10 @@ XrtComputationClient::XrtComputationClient(
                << "/replica:0/task:" << worker_target.first.task_no;
   }
   TF_VLOG(1) << "XRT default device: " << options_.default_device;
-  MaybeCreateLocalService(options_);
+  // Only one localServer should be created.
+  if (options_.devices.count("CPU:0") > 0) {
```

Comment:
For my understanding, why is this newly needed?

Reply:
For multiprocessing tpu setup, all process will try to call `MaybeCreateLocalService` but we only want to start one localServer. In a 8 TPU devices will be like

`CPU:0` + `TPU:0`
`TPU:1`
`TPU:2`
`TPU:3`
`TPU:4`
`TPU:5`
....

so I was making sure only the process with the `CPU:0` will try to start the localserver. .This feels like a hack and it break our python tests.. Will think more about it.. Do you have any suggestions?